src/test/datascience/kernelLauncher.functional.test.ts

```diff
@@ -41,7 +44,8 @@ suite('DataScience - Kernel Launcher', () => {
         snapshot = takeSnapshot();
     });
 
-    setup(async () => {
+    setup(async function () {
+        traceInfo(`Start Test ${this.currentTest?.title}`);
```

Comment:
Increased timeout & added logging to determine why this test was failing (timeout error)